grafanalib/core.py

```diff
@@ -2585,11 +2592,20 @@ def to_json_data(self):
 
 @attr.s
 class Threshold(object):
-    """Threshold for a gauge
+    """Threshold for for panels
+    (https://grafana.com/docs/grafana/latest/panels/thresholds/)
```

Comment:
I suppose that the JSON model for panel threshold is a bit different. It should contain `colorMode` with `ok`,  `warning`, `critical`, `custom` values; boolean `fill`; not null `value` and optional `fillColor` and `lineColor` for `custom` mode if the corresponding option is enabled.

Reply:
Thanks for the review, I will look at this towards the end of the week.

Reply:
I have added a separate class for Graph panels to support this